src/app/themes.tsx

```diff
@@ -4,58 +4,151 @@ import GlobalStyle from "@/app/global_styles";
-    backgroundColor: "#eeeeee",
-    primaryForegroundColor: "#000000",
-    primaryBackgroundColor: "#63a036",
-    secondaryForegroundColor: "#ffffff",
-    secondaryBackgroundColor: "#066940",
-    accentBackgroundColor: "#609fd3",
-    accentForegroundColor: "#000000",
-    surfaceForegroundColor: "#000000",
-    surfaceBackgroundColor: "#ffffff",
-    errorColor: "#ff624e",
-};
+const BLACK = "#000000";
+const WHITE = "#f2f2f2";
 
-const darkTheme: CustomTheme = {
-    foregroundColor: "#ffffff",
-    backgroundColor: "#1a1a1a",
-    primaryForegroundColor: "#ffffff",
-    primaryBackgroundColor: "#066940",
-    secondaryForegroundColor: "#000000",
-    secondaryBackgroundColor: "#63a036",
-    accentBackgroundColor: "#1b629c",
-    accentForegroundColor: "#ffffff",
-    surfaceForegroundColor: "#ffffff",
-    surfaceBackgroundColor: "#161414",
-    errorColor: "#ff624e",
+export const lightTheme: DefaultTheme = {
+    light: true,
+    main: {
+        background: ["#fdfdfd", "#f6f6f6", "#eeeeee", "#d0d0d0"],
+        foreground: ["#2d2d2d", "#2d2d2d", "#2d2d2d", BLACK],
+        largeForeground: ["#2d2d2d", "#2d2d2d", "#2d2d2d", BLACK],
+        lighterForeground: ["#666666", "#666666", "#666666", "#3b3b3b"],
+        border: "#d5d5d5",
+    },
+    primary: {
+        background: ["#e7f5e9", "#7fd495", "#02985a", "#05663f", "#222f28"],
+        foreground: [BLACK, BLACK, BLACK, WHITE, WHITE],
+        largeForeground: [BLACK, BLACK, WHITE, WHITE, WHITE],
+    },
+    accent: {
+        background: "#1b385f",
+        foreground: WHITE,
+        largeForeground: WHITE,
+    },
+    error: "#ff624e",
+    shadow: "#e2e2e2",
+    rainbow: {
+        lightRed: {
+            background: "#fa9189",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkRed: {
+            background: "#fc4942",
+            foreground: BLACK,
+            largeForeground: WHITE,
+        },
+        lightOrange: {
+            background: "#fcae7c",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkOrange: {
+            background: "#fd7e2a",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        lightYellow: {
+            background: "#ffd868",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkYellow: {
+            background: "#ffbb00",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        lightGreen: {
+            background: "#b4f5b3",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkGreen: {
+            background: "#5fa881",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        lightBlue: {
+            background: "#acd5df",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkBlue: {
+            background: "#7bbafc",
```

Comment:
Not sure if I've just misunderstood the meanings of this, but surely dark blue would want a white foreground?

Reply:
The dark blue isn't really that dark haha - it is a pantone color